metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -29,26 +29,123 @@ def initialize
         @instrument_registry = {}
       end
 
+      # Counter is a synchronous Instrument which supports non-negative increments.
```

Comment:
What do we think about hyperlinking to the docs? I noticed some of them provide additional links and examples that may be useful. Could go either way!

```suggestion
      # {https://opentelemetry.io/docs/specs/otel/metrics/api/#counter Counter} is a synchronous Instrument which supports non-negative increments.
```